api.bs

```diff
@@ -1951,37 +1957,184 @@ To <dfn noexport>parse a `Save-Impression` header</dfn> given a [=header value=]
      :: |histogramIndex|
-     : {{AttributionImpressionOptions/matchValue}}
-     :: |matchValue|
-     : {{AttributionImpressionOptions/conversionSites}}
-     :: |conversionSites|
-     : {{AttributionImpressionOptions/conversionCallers}}
-     :: |conversionCallers|
-     : {{AttributionImpressionOptions/lifetimeDays}}
-     :: |lifetimeDays|
-     : {{AttributionImpressionOptions/priority}}
-     :: |priority|
+1.  If |dict|["<code>[=save-impression/conversion-sites=]</code>"] [=map/exists=]:
+     1.  Let |conversionSites| be its [=map/value=].
+     1.  If |conversionSites| is not an [=structured header/inner list=], or if any of
+         |conversionSites|' [=list/items=] is not a [=structured header/string=],
+         return an error.
+     1.  Set |opts|.{{AttributionImpressionOptions/conversionSites}} to |conversionSites|.
+1.  If |dict|["<code>[=save-impression/conversion-callers=]</code>"] [=map/exists=]:
+     1.  Let |conversionCallers| be its [=map/value=].
+     1.  If |conversionCallers| is not an [=structured header/inner list=], or if any of
+         |conversionCallers|' [=list/items=] is not a [=structured header/string=],
+         return an error.
+     1.  Set |opts|.{{AttributionImpressionOptions/conversionCallers}} to |conversionCallers|.
+1.  If |dict|["<code>[=save-impression/match-value=]</code>"] [=map/exists=]:
+     1.  Let |matchValue| be its [=map/value=].
+     1.  If |matchValue| is not a non-negative [=structured header/integer=], return an error.
+     1.  Set |opts|.{{AttributionImpressionOptions/matchValue}} to |matchValue|.
+1.  If |dict|["<code>[=save-impression/lifetime-days=]</code>"] [=map/exists=]:
+     1.  Let |lifetimeDays| be its [=map/value=].
+     1.  If |lifetimeDays| is not a positive [=structured header/integer=], return an error.
+     1.  Set |opts|.{{AttributionImpressionOptions/lifetimeDays}} to |lifetimeDays|.
+1.  If |dict|["<code>[=save-impression/priority=]</code>"] [=map/exists=]:
+     1.  Let |priority| be its [=map/value=].
+     1.  If |priority| is not an [=structured header/integer=], return an error.
+     1.  Set |opts|.{{AttributionImpressionOptions/priority}} to |priority|.
+1.  Return |opts|.
 
 </div>
 
+## Measuring Conversions ## {#http-api-conversions}
+
+\`<dfn http-header><code>Measure-Conversion</code></dfn>\` is a
+[=structured header/dictionary|Dictionary Structured Header=]
+set on a response requesting that the user agent invoke the
+<a method for=Attribution>measureConversion()</a> API.
+
+<div class=example id=ex-measure-conversion-header>
+This is the HTTP equivalent of <a href=#ex-measure-conversion>the JavaScript `measureConversion` example</a>:
+<xmp highlight=http>
+Measure-Conversion: aggregation-service="https://aggregator.example/tee", histogram-size=20, epsilon=1.0, lookback-days=14, impression-sites=("publisher.example" "other.example"), impression-callers=("ad-tech.example"), match-values=(2), credit=(0.25 0.25 0.5), value=3, max-value=7
```

Comment:
The value of `aggregation-service` is not something that can be looked up through this API.  That makes this far less reliable than the JS API, which at least can adapt.

Another thing that needs to be noted clearly is that this doesn't let you have multiple `Measure-Conversion` fields.  If there are multiple conversions, then maybe a redirect chain will allow those to be logged.